From 9b6037d02745342eb68b564f61c34bd908957ca3 Mon Sep 17 00:00:00 2001
From: Jakub Ruzicka <jruzicka@redhat.com>
Date: Wed, 13 Aug 2014 16:02:22 +0200
Subject: [PATCH] Use oslo.sphinx instead of oslosphinx

---
 doc/source/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index d33aea2..e4d1b8f 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -88,7 +88,7 @@ gen_ref("v3", "Version 3 API Reference",
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'oslosphinx']
+extensions = ['sphinx.ext.autodoc', 'oslo.sphinx']
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
